modules/serverless-rn-1-31-0.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-31-0_{context}"]
+= Red Hat {ServerlessProductName} 1.31
+
+{ServerlessProductName} 1.31 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-31-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.10.
+* {ServerlessProductName} now uses Knative Eventing 1.10.
+* {ServerlessProductName} now uses Kourier 1.10.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.10.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.10.
+* The `kn func` CLI plug-in now uses `func` 1.11.
+
+* {ServerlessProductName} multi-tenancy with {SMProductShortName} is now available as a Technology Preview (TP) feature.
+
+* {FunctionsProductName} now supports the Go runtime as a Technology Preview (TP) feature.
+
+* Serverless Logic, which is available as a Technology Preview (TP) feature, has been updated.
++
+See link:https://openshift-knative.github.io/docs/docs/latest/serverless-logic/about.html[the Serverless Logic documentation] for usage instructions.
+
+* You can now configure a persistent volume claim (PVC) for an existing `PersistentVolume` object to use with a Serverless function.
+
+* When specifying Kourier for Ingress and using `DomainMapping`, the TLS for OpenShift Route is set to passthrough, and TLS is handled by the Kourier Gateway. Beginning with {ServerlessProductShortName} 1.31, it is possible to specify the enabled cipher suites on the side of the Kourier Gateway.
+
+* Integrating {SMProductName} with {ServerlessProductShortName} when Kourier is enabled is now deprecated. Use `net-istio` instead of `net-kourier` for Service Mesh integration.
++
+See the "Integrating {SMProductName} with {ServerlessProductShortName}" section for details.
+
+* A `PodDistruptionBudget` and a `HorizontalPodAutoscaler` have been added for the `3scale-kourier-gateway` deployment.
```

Comment:
Is there a noun that can follow `PodDistruptionBudget` and `HorizontalPodAutoscaler`? Not sure if "object" applies or something else